app/src/androidTest/java/io/github/polysmee/settings/UserInfoDataStoreTest.java

```diff
@@ -41,10 +43,11 @@ public static void setUp() throws Exception {
     }
 
     public static void testNameDatabase(String value){
+        sleep(1, SECONDS);
         new DatabaseUser(MainUserSingleton.getInstance().getId()).getName_Once_AndThen(name -> assertEquals(value, name ));
         sleep(1, SECONDS);
     }
```

Comment:
You must not use DatabaseUser. Instead, you should directly get the value from the database and use a Tasks.await(...) (see the others tests).